packages.yml

```
@@ -1,3 +1,3 @@
 packages:
   - package: dbt-labs/dbt_utils
-    version: [">=0.8.0", "<0.9.0"]
+    version: [">=0.9.0", "<2.0.0"]
```

```suggestion
    version: [">=0.8.0", "<2.0.0"]
```

Not doing any harm to keep 0.8.0 around - there are no breaking changes for this package